<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JKFallingBlocksMainWindow</class>
 <widget class="QWidget" name="JKFallingBlocksMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JKFallingBlocksMainWindow</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>LEVEL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labLevel">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="indent">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>SCORE:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labPoints">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="indent">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>LINES REMOVED:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labLinesRemoved">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="indent">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>229</width>
         <height>407</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string>&amp;START</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPause">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;PAUSE</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" rowspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>NEXT PIECE:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labNext">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="indent">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;u&gt;&lt;b&gt;Help:&lt;/b&gt;&lt;/u&gt;&lt;br&gt;&lt;br&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&lt;font size=&quot;+5&quot;&gt;&lt;tt&gt;&amp;larr;&lt;/tt&gt;&lt;/font&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp; move piece left&lt;br&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&lt;font size=&quot;+5&quot;&gt;&lt;tt&gt;&amp;rarr;&lt;/tt&gt;&lt;/font&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp; move piece right&lt;br&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&lt;font size=&quot;+5&quot;&gt;&lt;tt&gt;&amp;uarr;&lt;/tt&gt;&lt;/font&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp; rotate piece&lt;br&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&lt;font size=&quot;+5&quot;&gt;&lt;tt&gt;&amp;darr;&lt;/tt&gt;&lt;/font&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp; move piece down fast&lt;br&gt;
&amp;nbsp;&amp;nbsp;&amp;nbsp;&lt;tt&gt;SPACE&lt;/tt&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp; drop piece&lt;br&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="indent">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>306</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="JKFallingBlocksFrame" name="FallingBlocks">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>250</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>JKFallingBlocksFrame</class>
   <extends>QFrame</extends>
   <header>jkfallingblocksframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
